homeassistant/components/deconz/logbook.py

```diff
@@ -25,9 +125,19 @@ def async_describe_deconz_event(event: Event) -> dict:
             hass, event.data[ATTR_DEVICE_ID]
         )
 
+        if deconz_event.device.modelid not in REMOTES:
+            return {
+                "name": f"{deconz_event.device.name}",
+                "message": f"fired event '{event.data[CONF_EVENT]}'.",
+            }
+
+        action, interface = _get_device_event_description(
+            deconz_event.device.modelid, event.data[CONF_EVENT]
+        )
+
         return {
             "name": f"{deconz_event.device.name}",
-            "message": f"fired event '{event.data[CONF_EVENT]}'.",
+            "message": f"'{ACTIONS[action]}' event for '{INTERFACES[interface]}' was fired.",
```

Comment:
We likely need to make these translatable in the future

Reply:
Yes I know, talked a bit about this with Paulus before starting the previous logbook PR. Seems spacegaier might start work on that